### ISSUE TITLE
Keyboard navigation not working in sidebar

### ISSUE THREAD

**Issue by GVodyanov** [nextcloud/calendar#7930]
### Steps to reproduce

Issues:
- If using tab navigation, focus gets trapped on the currently selected calendar in the calendar list
- Checkboxes in the calendar list cannot be modified through keyboard navigation

There may be other instances of these issues

### Expected behavior

The app should be fully useable with keyboard navigation

### Actual behavior

The app isn't fully useable with keyboard navigation

### Web server

None

### Database engine version

None

### PHP engine version

None

### Updated from an older installed version or fresh install

None

### List of activated apps

```Text

```

### Nextcloud configuration

```Text

```

### Web server error log

```Text

```

### Log file

```Text

```

### Browser log

```Text

```

**Comment by GVodyanov** [comment]
This was discovered to be a Nextcloud vue issue https://github.com/nextcloud-libraries/nextcloud-vue/issues/8205
